content/en/docs/tasks/job/pod-failure-policy.md

```diff
@@ -242,6 +242,65 @@ kubectl delete jobs/job-pod-failure-policy-config-issue
 
 The cluster automatically cleans up the Pods.
 
+## Using Pod Failure Policy to avoid unnecessary Pod retries per index
+
+To avoid unnecessary Pod restarts per index, you can use the Pod failure policy and
+Backoff limit per index features. The following section shows how to use these features:
+
+1. Save the following manifest as `job-backoff-limit-per-index-failindex.yaml`:
+
+   {{% code_sample file="/controllers/job-backoff-limit-per-index-failindex.yaml" %}}
+
+1. Apply the manifest:
+
+   ```sh
+   kubectl create -f job-backoff-limit-per-index-failindex.yaml
```

Comment:
```suggestion
   kubectl create -f  https://k8s.io/examples/controllers/job-backoff-limit-per-index-failindex.yaml
```

Reply:
This makes sense, but we should probably be consistent with the rest of the doc. Would you prefer to update it consistently in this PR, or keep as is and leave to a follow up?

Reply:
Addressed tentatively in the latest commit: https://github.com/kubernetes/website/pull/49811/commits/d1eb544f18c29ae024d7ef0bf4426973061d5c54
It makes the diff bigger, but I like making the sections consistent. PTAL.

Reply:
Ok, fine to improve the other sections too.